app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file, new_file = command.split()[1:3]
```

Comment:
Perform split operation and save result into variable. Check that lenght is exactly 3. Check that first item is 'cp'